core/static/js/features.js

```diff
@@ -34,6 +35,10 @@ var FeatureStyles = FeatureStyles || {
                     }
 
                 }
+                if ( Modernizr.touch ) {
+                    $('video').hide();
```

Comment:
this is kind of heavy handed, it would be nice to limit it by a CSS class to future proof us.